bql/planner/data_access_test.go

```diff
@@ -562,38 +560,43 @@ func TestDataAccessTripleToRowObjectBindingsDroping(t *testing.T) {
 				OTypeAlias: "?o",
 				OIDAlias:   "?id",
 			},
-			bc: &table.Cell{N: n},
-			ac: &table.Cell{N: n},
-			tc: &table.Cell{S: table.CellString(n.Type().String())},
-			ic: &table.Cell{S: table.CellString(n.ID().String())},
+			oBinding:   &table.Cell{N: n},
+			oAlias:     &table.Cell{N: n},
+			oTypeAlias: &table.Cell{S: table.CellString(n.Type().String())},
+			oIDAlias:   &table.Cell{S: table.CellString(n.ID().String())},
 		},
 		{
 			t: n.String() + "\t" + p.String() + "\t" + p.String(),
 			cls: &semantic.GraphClause{
 				OBinding:       "?o",
 				OAlias:         "?alias",
 				OIDAlias:       "?id",
-				OAnchorBinding: "?ts",
+				OAnchorBinding: "?anchorBinding",
 				OAnchorAlias:   "?o",
 			},
-			bc:  &table.Cell{P: p},
-			ac:  &table.Cell{P: p},
-			ic:  &table.Cell{S: table.CellString(string(p.ID()))},
-			tsc: &table.Cell{T: ts},
-			atc: &table.Cell{T: ts},
+			oBinding:       &table.Cell{P: p},
+			oAlias:         &table.Cell{P: p},
+			oIDAlias:       &table.Cell{S: table.CellString(string(p.ID()))},
+			oAnchorBinding: &table.Cell{T: ta},
+			oAnchorAlias:   &table.Cell{T: ta},
 		},
 	}
+
 	for _, entry := range testTable {
```

Comment:
I'm not sure what I 'm missing here, but you seem to use only `entry.t`, and `entry.cls`. Where do you use the cells in the entry?

Reply:
You are right. I believe that when this part of the code was first developed just a copy-paste was done from the setup of the test function above.
I have just removed what is not used there in `TestDataAccessTripleToRowObjectBindingsDropping` to make it cleaner. 👍